applications/callflow/src/module/cf_branch_property.erl

```diff
@@ -0,0 +1,92 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2013-2016, 2600Hz INC
+%%% @doc
+%%%
+%%% "data":{
+%%%   "action":"branch", % the only one action supported currently
```

Comment:
Yes, it just marks that it's reserved for future use.
Future possibilities - branch_and_update or stuff like that.